edu.stanford.Almond.json

```diff
@@ -0,0 +1,272 @@
+{
+    "app-id" : "edu.stanford.Almond",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.28",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.gfortran-62"
+    ],
+    "command" : "/app/bin/edu.stanford.Almond",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=home",
+        "--filesystem=host",
```

Comment:
You don't need both `host` and `home`.

Reply:
Are you saying that host subsumes home? Is it documented somewhere?

Reply:
Not super clearly documented but host == can access all files. http://docs.flatpak.org/en/latest/sandbox-permissions-reference.html#filesystem-permissions

Reply:
"The entire host file system, except for directories that are handled specially by Flatpak. In particular, this shares /home, /media, /opt, /run/media and /srv if they exist."

https://jlk.fjfi.cvut.cz/arch/manpagesman/extra/flatpak/flatpak-metadata.5.en#[Context]